### PR TITLE
Disable fade transition

### PR DESCRIPTION
**Pull request by Dreamescaper**
As agreed in #30, removed fade transition, which allowed me to use AerielDream on my Sony TV.

### CODE REVIEW

File: app/src/main/java/com/codingbuffalo/aerialdream/ExoPlayerView.java [hunk 1]
```diff
@@ -25,7 +25,7 @@
 import com.google.android.exoplayer2.upstream.cache.SimpleCache;
 import com.google.android.exoplayer2.video.VideoListener;
 
-public class ExoPlayerView extends TextureView implements MediaController.MediaPlayerControl, VideoListener, Player.EventListener {
+public class ExoPlayerView extends SurfaceView implements MediaController.MediaPlayerControl, VideoListener, Player.EventListener {
```
Comment (cachapa) [inline app/src/main/java/com/codingbuffalo/aerialdream/ExoPlayerView.java:28]: Why the change to a surface view?

**Review by cachapa — COMMENTED** [review]
Thanks for the PR. I just the one remark about the change to a SurfaceView, otherwise it looks good.

File: app/src/main/java/com/codingbuffalo/aerialdream/ExoPlayerView.java (hunk 1 above)
Comment (Dreamescaper) [reply app/src/main/java/com/codingbuffalo/aerialdream/ExoPlayerView.java:28]: I don't remember the exact error (since I've made this change quite a while ago), l but 4K playback wasn't working for me until I changed it to SurfaceView.

Something with hardware acceleration not working with TextureView, as far as I remember.

Comment (cachapa) [reply app/src/main/java/com/codingbuffalo/aerialdream/ExoPlayerView.java:28]: Ok, thanks. SurfaceViews are (or were?) generally more efficient, but do not support transformations like transparency which your patch removes anyway.

Ok I'm merging this and will publish an update soon.

**Comment by JBallin** [comment]
Thank you both for fixing this! 4k works on my tv now (playback is smooth - had choppines before, and the transitions work just fine - flashes a black screen) 🎉 

I have just 1 other issue with the app, that I was hopeful would be fixed as part of #30 - but it's still not working unfortunately - opened #34
